linkup-cli/src/commands/deploy/cf_deploy.rs

```diff
@@ -457,11 +504,13 @@ export default {
         };
 
         let resources = test_resources();
-        let _mocks = mock_cloudflare_endpoints(&resources);
+
+        let mut mock_server = mockito::Server::new_async().await;
+        let client = test_client(mock_server.url());
+        let _mocks = mock_cloudflare_endpoints(&mut mock_server, &resources).await;
```

Comment:
We aren't really returning anything from `mock_cloudflare_endpoints`, right? I guess we can remove the `let _mock`